docs/patterns/health-endpoint-monitoring.md

```diff
@@ -193,20 +120,19 @@ The conditions you can monitor vary depending on the hosting mechanism you choos
 
```

Comment:
Can you update the lines just before this comment that mention "Cloud Services" and "Mobile Services" as well.  We should freshen that paragraph up.  Sorry I can't add this comment right on that line, GitHub UI won't let me.

Reply:
you can probably just remove the whole (note) part.  If you leave it like you have it, you need to fix your casing.